### DISCUSSION TITLE
EADDRINUSE?

### DISCUSSION THREAD

**Discussion by AgentCochese** [GitSquared/edex-ui#1151] [Questions] [answered]
![EAADRS](https://user-images.githubusercontent.com/69368965/128919532-7a3e134f-4f14-4f60-b096-886c58d59555.png)

Hey all, trying to get this running on a high end HP ZBook Core i9 with 64GB RAM (Windows 10 Enterprise) and I get the error: 

eDEX-UI crashed
listen EADDRINUSE: address already in use :::3000

I searched online for a fix and I see it may be a conflict with another resource but I can't solve it.

Thoughts?

**Comment by GitSquared** [comment, marked as answer]
You likely have something running on the `3000` port - look for local development servers & related tooling... Relaunch edex once the port is free
